certified-connectors/Taqnyat SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,393 @@
+            "x-ms-summary": "Success response",
+            "schema": {
+              "$ref": "#/definitions/SendSMSSuccessResponseBody"
+            }
+          },
+          "400": {
+            "description": "Response to failed send sms.",
+            "x-ms-summary": "Failed response",
+            "schema": {
+              "$ref": "#/definitions/FailureResponseBody"
+            }
+          },
+          "default": {
+            "description": "Response to failed send sms.",
+            "x-ms-summary": "Failure response",
+            "schema": {
+              "$ref": "#/definitions/FailureResponseBody"
+            }
+          }
+        },
+        "summary": "Send SMS",
+        "description": "Send SMS To Recipients Mobile",
+        "operationId": "SendSMS",
+        "parameters": [
+          {
+            "name": "Request body",
+            "in": "body",
+            "description": "Request body.",
+            "schema": {
+              "$ref": "#/definitions/SendSMSRequestBody"
+            }
+          }
+        ]
+      }
+    },
+    "/v1/messages/senders": {
+      "get": {
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "201": {
+            "description": "Response to successful get senders.",
+            "x-ms-summary": "Success response",
+            "schema": {
+              "$ref": "#/definitions/GetSendersSuccessResponseBody"
+            }
+          },
+          "400": {
+            "description": "Response to failed get senders.",
+            "x-ms-summary": "Failed response",
+            "schema": {
+              "$ref": "#/definitions/FailureResponseBody"
+            }
+          },
+          "default": {
+            "description": "Response to failed get senders.",
+            "x-ms-summary": "Failure response",
+            "schema": {
+              "$ref": "#/definitions/FailureResponseBody"
+            }
+          }
+        },
+        "summary": "Sender Names",
+        "description": "Get Your Verified Sender Names",
+        "operationId": "GetSenders",
+        "parameters": []
+      }
+    },
+    "/account/balance": {
+      "get": {
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "201": {
+            "description": "Response to successful get account balance.",
+            "x-ms-summary": "Success response",
+            "schema": {
+              "$ref": "#/definitions/GetAccountBalanceSuccessResponseBody"
+            }
+          },
+          "400": {
+            "description": "Response to failed get account balance.",
+            "x-ms-summary": "Failed response",
+            "schema": {
+              "$ref": "#/definitions/FailureResponseBody"
+            }
+          },
+          "default": {
+            "description": "Response to failed get account balance.",
+            "x-ms-summary": "Failure response",
+            "schema": {
```

Comment:
@TaqnyatNocTeam The 'default' response should not have schema definition. Schemas should be defined on expected responses only. Please remove the schema.